packages/hamburger-react/src/Hamburger.tsx

```diff
@@ -20,18 +22,24 @@ export const Hamburger = ({
     onClick,
     inverted = false,
     description = "Hovedmeny",
-    className = "",
+    className,
     actionLabel,
 }: Props) => {
     const componentClassname = classnames(
         "jkl-hamburger",
         {
             "jkl-hamburger--is-open": isOpen,
             "jkl-hamburger--inverted": inverted,
+            "jkl-hamburger--label-before": actionLabel && actionLabel.position !== "after",
+            "jkl-hamburger--label-after": actionLabel?.position === "after",
```

Comment:
```suggestion
            "jkl-hamburger--label-before": actionLabel?.position === "before",
            "jkl-hamburger--label-after": actionLabel && actionLabel.position !== "before",
```

Se svaret mitt på @Mikaila94 sin kommentar for hvorfor

Reply:
Stemmer, når du sier det så husker jeg at det var en grunn til å gjøre det sånn ja...